cms/models/placeholdermodel.py

```diff
@@ -299,4 +306,81 @@ def actions(self):
             self._actions_cache = getattr(field, 'actions', PlaceholderNoAction())
         return self._actions_cache
 
+    def get_cache_expiration(self, request, response_timestamp):
+        """
+        Returns the number of seconds (from «response_timestamp») that this
+        placeholder can be cached. This is derived from the plugins it contains.
+
+        This method must return: EXPIRE_NOW <= int <= MAX_EXPIRATION_IN_SECONDS
+
+        :type request: HTTPRequest
+        :type response_timestamp: datetime
+        :rtype: int
+        """
+        lang = getattr(request, 'lang', None)
+
+        min_ttl = MAX_EXPIRATION_TTL
+        for plugin_item in self.get_plugins(lang):
+            instance, plugin = plugin_item.get_plugin_instance()
```

Comment:
That would seem logical, but I can't seem to demonstrate that this is quite correct. I've tried with both caching enabled and disabled but the number of queries seem does grow with the number of plugins on the page when in edit mode, but the exact same number of queries are executed with `return 0` inserted before like 323 above (disabling the line in question), and only when the user is logged-in and in edit mode. Rolling back to `develop` and trying again, exhibits the same behavior.

Even if I was adding these queries, which I don't think I am, they are impacting performance by about 225 µsecs per plugin on my local system. (NOTE: that is **micro**seconds, not **milli**seconds).

Considering the fact that this line is clearly _not_ adding any queries, I think its is worth it to keep them in, to maintain the same(-ish) signature as the render() method.